src/pages/signin/ValidateCodeForm/BaseValidateCodeForm.js

```diff
@@ -98,26 +95,11 @@ function BaseValidateCodeForm(props) {
     }, [props.account.isLoading, props.session.autoAuthState, hasError]);
 
     useEffect(() => {
-        if (!inputValidateCodeRef.current || prevIsVisible || !props.isVisible || !canFocusInputOnScreenFocus()) {
+        if (!inputValidateCodeRef.current || !canFocusInputOnScreenFocus()) {
             return;
         }
         inputValidateCodeRef.current.focus();
-    }, [props.isVisible, prevIsVisible]);
-
-    useEffect(() => {
-        if (!inputValidateCodeRef.current) {
-            return;
-        }
-
-        if (!validateCode) {
-            return;
-        }
-
-        // Clear the code input if magic code valid
-        if (prevIsVisible && !props.isVisible) {
-            setValidateCode('');
-        }
-    }, [props.isVisible, prevIsVisible, validateCode]);
+    }, []);
```

Comment:
I know this was from the previous code, but we don't have to pass `inputValidateCodeRef.current` and `canFocusInputOnScreenFocus` to the dependency array?

Reply:
ah I just checked. ESLint doesn't give warnings for these values.

thanks you for your explanation @huzaifa-99 🙇